drivers/flash/nrf_qspi_nor.c

```diff
@@ -564,7 +576,20 @@ static int qspi_nrfx_configure(const struct device *dev)
 		}
 	}
 
-	return 0;
+	if (INST_0_4BA != 0) {
```

Comment:
Maybe we should add LOG_DBG that would be stating that we got into 4-byte address mode.

Reply:
done